examples/point_robot_reward.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+from urdfenvs.urdf_common.urdf_env import UrdfEnv
+from urdfenvs.robots.generic_urdf import GenericUrdfReacher
+from urdfenvs.sensors.full_sensor import FullSensor
+from urdfenvs.scene_examples.goal import goal1
+from urdfenvs.urdf_common.reward import Reward
+
+
+N_STEPS=1000
+
+
+class InverseDistanceDenseReward(Reward):
+    def calculateReward(self, observation: dict) -> float:
+        goal = observation['robot_0']['FullSensor']['goals'][1]['position']
+        position = observation['robot_0']['joint_state']['position']
+        return 1.0/np.linalg.norm(goal-position)
+
+
+robots = [
+    GenericUrdfReacher(urdf="pointRobot.urdf", mode="vel"),
+]
+env= UrdfEnv(
+    dt=0.01,
+    robots=robots,
+    render=True,
+)
+env.add_goal(goal1)
+
+# %%
+sensor = FullSensor(['position'], ['position', 'size'], variance=0.0)
+env.add_sensor(sensor, [0])
+env.set_spaces()
+env.set_reward_calculator(InverseDistanceDenseReward())
```

Comment:
This is clearly a better implementation than inputting things into the class constructor.